src/app/commands/cmd_options.cpp

```diff
@@ -926,6 +926,18 @@ class OptionsWindow : public app::gen::Options {
       }
     }
 
+    // Get the extension information from the compressed
+    // package.json file.
+    const ExtensionInfo info =
+      App::instance()->extensions().getCompressedExtensionInfo(filename);
+    // Check if the filename corresponds to aseprite-default theme
+    if (base::string_to_lower(info.name) ==
+        Extensions::kAsepriteDefaultThemeName) {
```

Comment:
Probably ExtensionInfo needs a `bool` property like `defaultTheme` instead of the `themeId` (below more details).